CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.4.1
+- add Stop and Start task (contribution by to Sebastian Schuberth [#79](https://github.com/novoda/gradle-android-command-plugin/pull/79))
```

Comment:
Should read `contribution by [Sebastian Schuberth](https://github.com/sschuberth)` (with user link)